tests/ChatTests/Files.hs

```diff
@@ -787,7 +787,8 @@ testXFTPCancelRcvRepeat =
       bob ##> "/fr 1 ./tests/tmp"
       bob
         <### [ "saving file 1 from alice to ./tests/tmp/testfile_1",
-               "started receiving file 1 (testfile) from alice"
+               "started receiving file 1 (testfile) from alice",
+               StartsWith "chat db error: SERcvFileNotFoundXFTP"
```

Comment:
this test always had error but a different one - CENoRcvFileUser, it was muted by log level. Error happens because agent reports next downloaded chunk when cancel happened in progress